xml/System.Security.Cryptography/FromBase64Transform.xml

```diff
@@ -225,15 +231,8 @@
       </ReturnValue>
       <Docs>
         <summary>Gets a value that indicates whether multiple blocks can be transformed.</summary>
-        <value>Always <see langword="false" />.</value>
-        <remarks>
-          <format type="text/markdown"><![CDATA[  
-  
-## Remarks  
- In the current implementation, only one block can be transformed at a time, so this property is always `false`.  
-  
- ]]></format>
-        </remarks>
+        <value>Always <see langword="true" />.</value>
+        <remarks>To be added.</remarks>
```

Comment:
Copied from `CanReuseTransform`. Is this correct?

Reply:
No, it's not correct:

>    public int InputBlockSize => 1;
       public int OutputBlockSize => 3;
       public bool CanTransformMultipleBlocks => false;
       public virtual bool CanReuseTransform => true;